src/sass/buttons/_base.scss

```diff
@@ -165,11 +165,13 @@
 }
 
 .#{$prefix}-button--small {
+  font-size: $ts-14;
   height: $spacing-md;
   line-height: 1.1;
   vertical-align: bottom;
   padding-right: $spacing-xs;
   padding-left: $spacing-xs;
+  padding-top: $spacing-xxs * 0.5;
```

Comment:
I think you're right in the particular example we have in the fractal site with the text being "Primary" this padding does help make the text appear more vertically centered optically. But, I worry that with different words that have various combinations of ascenders and descenders it could have the opposite affect making it look like there's more space on top than bottom.

See this screenshot below. The example on the right with a combination of uppercase letters and those with descenders ("y" and "g") look off center with the small amount of top padding applied.

![image](https://user-images.githubusercontent.com/5590034/154706885-2c7265f7-563e-47a4-b23b-5a99773e3b4d.png)